create-service-and-ensure-existence.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash -e
```

Comment:
I think the filename should be: `ensure-service-and-update-repo-metadata.sh`

Reply:
Also, please add a section to README to describe how to use this script. E.g. from within `circleci.yml` config.

Reply:
> I think the filename should be: `ensure-service-and-update-repo-metadata.sh`

msat-set-service-repo-url-metadata.sh